config/brakeman.ignore

```diff
@@ -1287,7 +1287,7 @@
       "warning_code": 123,
       "fingerprint": "715ee6d743a8af33c7b930d728708ce19c765fb40e2ad9d2b974db04d92dc7d1",
       "check_name": "EOLRuby",
-      "message": "Support for Ruby 3.1.6 ends on 2025-03-31",
+      "message": "Support for Ruby 3.4.1 ends on 2025-03-31",
```

Comment:
this shouldn't change. But we should remove this from GH ignore and re-run brakeman